tools/font-subset/test.py

```diff
@@ -58,7 +61,7 @@ def RunCmd(cmd, codepoints, fail=False):
     stderr=subprocess.PIPE,
     cwd=SRC_DIR
   )
-  stdout_data, stderr_data = p.communicate(input=' '.join(codepoints))
+  stdout_data, stderr_data = p.communicate(input=' '.join(codepoints).encode())
```

Comment:
`.encode` is needed for python3.

Reply:
You can also pass `universal_newlines=True` to `subprocess.Popen`.

Reply:
In addition to .encode, or instead of it?

Reply:
Ahh I see instead of it.